tests_single/feature_test.go

```diff
@@ -56,6 +56,7 @@ func TestRunFeatureSuite(t *testing.T) {
 			"TestVl3_dns",
 			"TestVl3_scale_from_zero",
 			"TestVl3_lb",
+			"TestVl3_basic",
```

Comment:
Note: this test has beed moved to the basic suite

Reply:
Thanks, fixed